setup.py

```diff
@@ -80,6 +82,7 @@
         "kerberos": kerberos_require,
         "sqlalchemy": sqlalchemy_require,
         "tests": tests_require,
+        "secure-local-storage": localstorage_require,
```

Comment:
does anyone have better names? `oauth-cache`/`oauth-keyring`?

Reply:
What do you think of `external-authentication-token-cache`? 

Would be similar to the [JDBC driver settings](https://trino.io/docs/current/installation/jdbc.html#parameter-reference).  Always helps to keep things as consistent as possible over multiple drivers.

Reply:
agreed, that seems a bit long but is very clear about what it does. @ebyhr your input would be helpful.

Reply:
Long name looks no problem because the name won't appear in the scripts frequently. I have no strong opinion though.

Reply:
@mdesmet Let's go with `external-authentication-token-cache` like the JDBC driver. Looks good to merge otherwise.